.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+sudo: required
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+node_js:
+  - "7"
+before_install:
+  - npm setup
+install:
+  - webpack
```

Comment:
Pardon me, is webpack will installed automatically ? 
I don't see any travis command to enable this ?

Reply:
Sorry there's typo in the install command. You can refer to the newest one

Reply:
When you run yarn setup, it will automatically installing the webpack